benchmarks/populate_into_db.py

```diff
@@ -0,0 +1,160 @@
+import argparse
+import os
+import sys
+
+import pandas as pd
+import psycopg2
+import psycopg2.extras
+
+
+FINAL_CSV_FILENAME = "collated_results.csv"
+# https://github.com/huggingface/transformers/blob/593e29c5e2a9b17baec010e8dc7c1431fed6e841/benchmark/init_db.sql#L27
+BENCHMARKS_TABLE_NAME = "benchmarks"
+MEASUREMENTS_TABLE_NAME = "model_measurements"
+
+
+def _init_benchmark(conn, branch, commit_id, commit_msg):
+    metadata = {"metadata": f"diffusers-{commit_id}"}
```

Comment:
@McPatate getting: https://github.com/huggingface/diffusers/actions/runs/16043615394/job/45269816792#step:10:22

[From](https://github.com/huggingface/transformers/blob/8178c43112295bf8c4ef04c667efbbbfd34b8bca/benchmark/llama.py#L48) what I understand, I _think_ I am doing things as expected here. Could you comment?

Reply:
You need to do the following:
```py
from psycopg2.extensions import register_adapter
from psycopg2.extras import Json

register_adapter(dict, Json)
```